### DISCUSSION TITLE
Quay distributed tracing

### DISCUSSION THREAD

**Discussion by michaelalang** [quay/quay#3375] [General]
Asking for everyone's oppinion on Distributed tracing integration for Quay. Since I am doing a Breakout session at Red Hat One and want to show-case real-life-products in the integration I did build a custom image of Quay providing such functionality.
If there's enough people interessted to see that, I will also show-case it in the Community call accordingly.

**Comment by dmesser** [comment]
@michaelalang Sounds like a great idea.

**Reply by michaelalang** [reply]
Ack I will show this to the BU after RH1 as my content is classified confidential (as anyother) until the event has passed.
I am also raising two more discussions were OTEL helped identifying how we can improve Quay code ...

Thanks for your response and happy new year !
